test/build/50_ch-image.bats

```diff
@@ -41,6 +41,53 @@ EOF
     echo "$output"
     [[ $status -eq 0 ]]
     [[ $output = *'verbose level: 1'* ]]
+
+    # unset debug in preparation for “--quiet” tests
+    unset CH_IMAGE_DEBUG
+
+    # test gestalt logging
+    run ch-image gestalt logging
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output = *"info"* ]]
+    [[ $output = *'warning: warning'* ]]
+    [[ $output = *'error: error'* ]]
+
+    # quiet level 1
+    run ch-image gestalt -q logging
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output != *"info"* ]]
+    [[ $output = *'warning: warning'* ]]
+    [[ $output = *'error: error'* ]]
+
+    # quiet level 2
+    run ch-image build --force seccomp -t tmpimg -qq -f - . << 'EOF'
+FROM almalinux:8
+RUN dnf install -y --setopt=install_weak_deps=false openssh-clients \
+ && dnf clean all
```

Comment:
Use explicit redirection on toy commands like `echo` rather than a big messy command like `dnf`. E.g.:
```suggestion
FROM alpine:3.17
RUN echo 'this is stdout'
RUN echo 'this is stderr' 1>&2
```